src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -177,5 +178,11 @@ public void Handle(ObservedDataRemovedFromAnalysableEvent eventToHandle)
       {
          updateOutputMappingList();
       }
+
+      public void Handle(SimulationOutputSelectionsChangedEvent eventToHandle)
```

Comment:
This all looks good

But where it the event actually thrown?
This is also a core UI and presenter stuff

Reply:
It will be right after this line of code
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/eb50ba0d6970bf7286cc7452e6048aecb344eb39/src/PKSim.Core/Services/InteractiveSimulationRunner.cs#L56

Reply:
not sure about this. This will have to be done in MoBi as well
Ihave the feeling this could be in core

Reply:
I tried in MoBi and it already refreshes somehow. But there is a similar spot for it in MoBi too.

Reply:
There is a common presenter in Core, but this little section of code in PKSim means only PKSim knows if the simulation will actually be updated.

https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/eb50ba0d6970bf7286cc7452e6048aecb344eb39/src/PKSim.Presentation/Presenters/Simulations/SimulationOutputSelectionPresenter.cs#L72-L73

Reply:
Same in MoBi

https://github.com/Open-Systems-Pharmacology/MoBi/blob/e52361e2094b3bfabf13a5d34428d0339d2ae28c/src/MoBi.Presentation/Presenter/OutputSelectionsPresenter.cs#L58-L59